### PR TITLE
Inteq risk management group, a neutral PMC faction

### PR DESCRIPTION
**Pull request by Ebin-Halcyon**
## About The Pull Request

[Things are still in it's early stages and i'm willing to let it cook longer if that's what everyone else wants]
adds in clothing + armor and a single weapon for a new faction thought up by @Drifter7371, Inteq risk management group; lore below. Also resprites the combat glove/combat boot sprites to not just match black gloves/jackboots

Obj sprites + new combat gloves/boots

image

Assembled getup of the few different types of people you get in the IRMG.
Digi jumpsuits are coming soon:tm: Digitigrade combat boots exist. 🦀 
image

The gun. Same exact stats as the basic AK47.

https://user-images.githubusercontent.com/81479835/153286491-3cb8315c-c714-4e00-8649-db008afaa1af.mp4

<details>
<summary>lore</summary>
<br>

Inteq Risk Management Group, or in short hand IRMG.
A private corporation that consists of Ex-Syndicate and Nanotrasen Personnel.
IRMG has formed after the corporate war,
The scars of Nuclear warfare having the most impact.
The equipment they have obtained was gained from an old Syndicate surplus, Some even stolen from supply drops. The equipment has been heavily modified for effectivity to serve in Private security operations. some of the more junior members complain about a hardline "no bullshit" policy, enforced both within their own ranks and the people they deal with. Rumors say that lethal punishment has been used to enforce this before, but regardless, it has contributed to IRMG's professional reputation. This certainly has sparked an Interesting conflict between IRMG and SolGov, but no hostilities have yet been made. IRMG has a moderate Reputation with the local sector for being top-quality military grade protection, often hired for body-guards or company security.
(All in all, they're a PMC with no real hostility to any other faction.)
</details>

## Why It's Good For The Game

New neutral factions, new combat boot sprites, new combat glove sprites, pretty neato.

## Changelog
:cl:
add: Inteq risk management group sprites and items
/:cl:

### CODE REVIEW

**Comment by Drifter7371** [comment]
Woooo Yeaaaah Woooo.

**Comment by Wildwestmars** [comment]
I know I'm a outsider but... it seems FAR to similar to the syndicate in color scheme, and it *red and blank contra villains* definitely doesn't feel like *neutral PMC*... also the fact this was used for a appeal for syndicate clothing on Sky-rats makes me feel this is simply just The Syndicate: 2 Electric Boogaloo.
If someone would want to play to play a mercenary, how would this appeal to them? I like the thought you wanna add neutral factions, just some thoughts I want you to consider.

**Comment by Drifter7371** [comment]
> I know I'm a outsider but... it seems FAR to similar to the syndicate in color scheme, and it _red and blank contra villains_ definitely doesn't feel like _neutral PMC_... also the fact this was used for a appeal for syndicate clothing on Sky-rats makes me feel this is simply just The Syndicate: 2 Electric Boogaloo. If someone would want to play to play a mercenary, how would this appeal to them? I like the thought you wanna add neutral factions, just some thoughts I want you to consider.


Well you see, even in the lore it states they still have a Bias, it also explains the equipment and it's origins.
The Bias they have will not affect how they react truly, but it will inspire the theme.
They're Ex-Syndicate, which would inspire some of the gear. The main theme of the gear is to be imposing and somewhat Threatening to make any hostile forces think twice about engaging.

**Comment by Apogee-dev** [comment]
> Well you see, even in the lore it states they still have a Bias, it also explains the equipment and it's origins. The Bias they have will not affect how they react truly, but it will inspire the theme. They're Ex-Syndicate, which would inspire some of the gear. The main theme of the gear is to be imposing and somewhat Threatening to make any hostile forces think twice about engaging.

at the same time a color scheme that's too similar to the syndicate factions can and will cause confusion among players (and, IC, among characters) as to their affiliation. If they aren't Syndicate, they shouldn't be this easy to confuse for Gorlex soldiers or ACLF agents. I'd strongly recommend changing the color scheme to something more faction-neutral, like olive, brown, or gold.

**Comment by Drifter7371** [comment]
> at the same time a color scheme that's too similar to the syndicate factions can and will cause confusion among players (and, IC, among characters) as to their affiliation. If they aren't Syndicate, they shouldn't be this easy to confuse for Gorlex soldiers or ACLF agents. I'd strongly recommend changing the color scheme to something more faction-neutral, like olive, brown, or gold.

I haven't exactly seen ACLF or Gorlex armor that matches what we're trying to put down here, So I think we're pretty much clear in that area, but as stated previously they are supposed to look similar to the Syndicate but not have an exact affiliation with them. As told, they are EX-SYNDICATE, Deserters in a sense. The gear they use is modified but still Syndicate Origins. Considerably they are branding the Syndicate in their own way. Not to mention that I think Olive or Gold would be hard to pull off, though if you could provide us a visual example of how it should look, That'd be helpful in a remake. No Guarantees we will follow-through unfortunately. Because this is infact what we are trying to go for. Something Similar, but not exactly alike.

**Comment by Apogee-dev** [comment]
> I haven't exactly seen ACLF or Gorlex armor that matches what we're trying to put down here, So I think we're pretty much clear in that area, but as stated previously they are supposed to look similar to the Syndicate but not have an exact affiliation with them. As told, they are EX-SYNDICATE, Deserters in a sense. The gear they use is modified but still Syndicate Origins. Considerably they are branding the Syndicate in their own way. Not to mention that I think Olive or Gold would be hard to pull off, though if you could provide us a visual example of how it should look, That'd be helpful in a remake. No Guarantees we will follow-through unfortunately. Because this is infact what we are trying to go for. Something Similar, but not exactly alike.

It's primarily the red-on-black color scheme, which, when it doesn't strongly resemble Syndie gear, instead resembles Nanotrasen security gear (the chest rig in particular is nearly identical to the Nanotrasen security chest rig, just with a slightly different shade of grey). I understand that this faction is supposed to be ex-syndicate, but from a gameplay perspective the color scheme is just... confusing. Red and black military-style equipment looks good but is already used by around four or so factions to begin with, if you count Syndicate sub-factions.
My personal recommendation would probably be an olive or brown shirt with either yellow or orange goggles & rig insignia. They're still warm colors adjacent to red but are visually distinct enough to avoid confusion with either the syndicate or nanotrasen security.

lore-wise it would probably be smart for an independent company, especially a newly-formed one, to avoid strong brand association with both the syndicate's combat arms and nanotrasen security anyway. it's bad branding.

**Comment by Ebin-Halcyon** [comment]
Oh I completely missed this. Yeah i'll just make a brown turtleneck + gold visor. See how that works.

**Comment by Oscari-pog** [comment]
I do not like that the lore is using names of real characters, played on skyrat. Injecting player characters into official lore is never a good idea.

Nick I. Talison, Hell, just noticed that the company name is Talison. No idea if you are the player, but if you play them on skyrat, that makes it even worse.

**Comment by Ebin-Halcyon** [comment]
> I do not like that the lore is using names of real characters, played on skyrat. Injecting player characters into official lore is never a good idea.
> 
> Nick I. Talison, Hell, just noticed that the company name is Talison. No idea if you are the player, but if you play them on skyrat, that makes it even worse.

I'm just the spriter, anything regarding names and lore can be brought up to @Drifter7371

**Comment by ZephyrTFA** [comment]
I'm amazed this passed checks given nothing else does

**Comment by Drifter7371** [comment]
Lore has been re-written, expect an update when Halc wakes up.

**Comment by Apogee-dev** [comment]
Just some proofreading. Most of the item descriptions have run-on sentences, you might want to break those up. The webbing also still has the old company name.

File: code/modules/clothing/head/helmet.dm [hunk 1]
```diff
@@ -440,3 +440,10 @@
 	for(var/X in actions)
 		var/datum/action/A = X
 		A.UpdateButtonIcon()
+
+/obj/item/clothing/head/helmet/swat/inteq
+	name = "inteq SWAT helmet"
+	desc = "A robust and spaceworthy helmet with an opaque burgundy visor, there is an insignia on the earpad with the letters 'I.C' on it."
```
Comment (Apogee-dev) [inline code/modules/clothing/head/helmet.dm]: Break up run-on sentences and update color descriptions e.g. "A robust and spaceworthy helmet with an opaque gold visor. There is an insignia on the earpad with the letters 'I.C' on it."

File: code/game/objects/items/storage/belt.dm [hunk 2]
```diff
@@ -698,3 +698,10 @@
 /obj/item/storage/belt/sabre/PopulateContents()
 	new /obj/item/melee/sabre(src)
 	update_icon()
+
+/obj/item/storage/belt/security/webbing/inteq
+	name = "talison corp webbing"
```
Comment (Apogee-dev) [inline code/game/objects/items/storage/belt.dm]: Out-of-date company name here

**Comment by Apogee-dev** [comment]
As a minor concern, the Inteq AKM is going to be overpowered simply because the AK-47 is also an absolute beast at the moment. It doesn't make sense to nerf its stats from the actual AK-47 unless you do the NT-AK thing and change its ammo caliber (and the NT-AK is also extremely strong right now, so that wouldn't change that much), but I just wanted to point that out for any mapper that would want to hand Inteq AKMs out like candy.

On a related note, an Inteq handgun (Perhaps a modification of the M1911?) would be a nice addition. Just please don't make it 9mm unless/until that ammo type gets a buff, because it's really weak right now.

**Comment by Drifter7371** [comment]
> As a minor concern, the Inteq AKM is going to be overpowered simply because the AK-47 is also an absolute beast at the moment. It doesn't make sense to nerf its stats from the actual AK-47 unless you do the NT-AK thing and change its ammo caliber (and the NT-AK is also extremely strong right now, so that wouldn't change that much), but I just wanted to point that out for any mapper that would want to hand Inteq AKMs out like candy.
> 
> On a related note, an Inteq handgun (Perhaps a modification of the M1911?) would be a nice addition. Just please don't make it 9mm unless/until that ammo type gets a buff, because it's really weak right now.

The AK is not a default weapon and will only be handed out for events or for officers. the default weapon for the faction would probably be the (M1911 as you've said as a sidearm) or the WT rifle. but we also have an EBR somewhere stashed in the files.

**Comment by WindowsErrors** [comment]
"Hello Apogee, nice helmet. How the hell are ya?"

Drifter has mistaken me for this dickhead Apogee. It seems logical because Apo also resides at Shiptest and in fact does the same exact thing I do. He also has a penchant for militaristic factions and middleweight ship designs. Apogee and I are even online during the same time slot, although my static has a slightly better flavor text.

He approaches where I'm sitting and stands over me.
"So how's the new ship design going, Apogee?" he responds in a half-hushed tone.
"It's alright" I respond distantly.
"Really?... it's interesting, it's not uhh... not great?"
I throw out a quick response. "Oh, well, you know"

He senses my disposition to the matter and changes the topic.
"So how's Cecilia? She's a great moth."
"Oh yea. I'm very lucky."
He can only let out an "Mhmmm."

From across the room, we both hear a "Hey, Apogee. Congratulations on the merged ship design." calling out to us, the sound of Wildwestmars and Oscari-pog entering the conference room.
The real Apogee perks up in the other corner of the room from Drifter and I with a "Thank you, Oscari!"

As they approach, Wildwestmars hits him with "Listen, Apogee. Sprites?"
Apogee reaches into his suit pocket, pulling out a branch and taking a couple thousand file changes out, handing it to him across the mahogany table in the middle of the conference room.
Apogee only says a simple "Call me." westmars takes a quick look at the card, glancing back up at Apogee, before he makes a quick "Wha'bout Saturday?"
Apogee starts walking out of the room with his spirit clearly lifted by the interaction while saying "No can do. I got an eight-thirty rez in the admin VC." Everyone else in the room stops for a second, almost in amazement at what Apogee has just said, all the while he only finishes up with a boastful "Great." while leaving.
Wildwestmars says "Admin voice chat on a Saturday night, how'd he swing that?", almost not expecting a response to it.
Oscari, next to him, responds with "I think he's lying."

With Apogee gone and no other competition, I figure it high time to reveal my own creation. I unholster my new branch and place my .dmi sprite card out onto the table.
I smartly begin with "New faction. What do you think?" as I slide them into the middle of the table for clear view. Everyone quickly leans over to get a good look. I can hear a "Whoa" and a "Very nice". I begin to smirk as one of them lets in more with a "Look at that..." I respond to all of them with "Picked it up from Dream Maker yesterday." and one of them compliments with "Good coloring." On a roll, I go to answer their unspoken questions. "That's Talison Company. And the lore style is something called 'Red & Black Military-Style'."

Oscari-pog counters me with "That's very cool, Halcyon, but that's nothing. Look at this." I can feel the blood rushing out of my head as he slaps his sprite card down onto the table, right next to mine, for all to see and compare. Wildwestmars immediately lets out "That is really nice", almost like he can't keep it in. Oscari finishes just as quickly. "Inteq Company. Main antagonist splinter group. Whadd'you think?"
I don't even think anything. I can't even think about drawing air into my lungs. All I can squeak out is a faint "Nice." Wildwestmars unconsciously rubs it in. "Jesus... huh, that is really super, how'd a nitwit like you get so tasteful?"
I can't believe that westmars prefers Oscari's faction to mine. I almost have to physically keep my mouth shut by putting my fist on my chin. Wildwestmars makes his proposition. "But wait. You ain't seen nothing yet." He reaches for his .dmi's while talking about them simultaneously. "Electric Boogaloo. Candy red. Ebony black."

I have to respond, even though I don't know how to. "Impressive." I make up blankly. "Very nice." I hem audibly. Their factions are better than mine. I divert their attention by requesting for us to see Apogee's faction. Everyone else slightly yet visibly slumps over, like they lost a certain amount of energy upon hearing my question. They pick up and put away their sprite cards without words. I wring my hands together as I only begin to realize what I've put upon myself. Wildwestmars reaches into his suit pocket to get Apogee's sprite card from earlier. He looks at it for a second, almost reluctantly choosing to swing it out into clear lamplight, giving me perfect vision.

The rest of the world shuts out of my mind in an instant. The subtle off-red coloring. The tasteful thickness of the charcoal finish. Oh my god, it even has a watermark. I can feel my heartbeat stuttering inside my ribcage. My grip on the card of .dmi's begins to weaken.

I can only hear Drifter by my side faintly. "Something wrong? Halcyon? You're sweating..."

**Comment by Kapu1178** [comment]
> 

based

**Comment by Ms-Mee** [comment]
> \<snip>

No way fanfic about Apogee my favorite shiptest comics character

**Comment by Drifter7371** [comment]
I am impressed yet concerned about this comment.
completely beyond words.

**Comment by Ebin-Halcyon** [comment]
>[snip]

Kino.

**Comment by Drifter7371** [comment]
Lore rewritten again, somewhat more interesting. (barely) (money back not guaranteed.)

**Comment by Wildwestmars** [comment]
I am both horrified and honored by what I read.

**Comment by Wildwestmars** [comment]
Also love the edits!

**Comment by Drifter7371** [comment]
> This pull request has conflicts, please resolve those before we can evaluate the pull request.

bruh.

**Comment by Ebin-Halcyon** [comment]
Well this _should_ be just about ready to merge in it's first state. More stuff like specialized ranks, clothing/gear for vanguards (hos), and IRMG specific guns should be able to be added at a later date. That or I keep this up longer and make it one huge dump off when I get everything Drifter wants in for sprites.

So like, yell at me to make it a draft again if you want this to cook a bit longer.

**Comment by Drifter7371** [comment]
Love to see it.

**Comment by Ebin-Halcyon** [comment]
Oh whoops -- wrong one, lol

**Comment by Apogee-dev** [comment]
Can we get some updated pictures with the newly added items and modified sprites?

**Comment by Ebin-Halcyon** [comment]
> Can we get some updated pictures with the newly added items and modified sprites?

Yeye, doing that now.

**Comment by Ebin-Halcyon** [comment]
All updated now.

**Comment by MarkSuckerberg** [comment]
Ready for merge? It seemed to do well in TMs.

**Comment by Ebin-Halcyon** [comment]
> Ready for merge? It seemed to do well in TMs.

Lemme push the turtleneck resprite and it'll be good to go.

**Pull request merged**